contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -772,13 +777,13 @@ contract JBETHPaymentTerminalStore {
 
     // Convert the _distributionRemaining to ETH.
     uint256 _ethDistributionRemaining = _distributionRemaining == 0
-      ? 0 // Get the current price of ETH. // A currency of 0 should be interpreted as whatever the currency being withdrawn is.
-      : _currency == 0
+      ? 0
+      : (_currency == JBCurrencies.ETH)
       ? _distributionRemaining
       : PRBMathUD60x18.div(_distributionRemaining, prices.priceFor(_currency, JBCurrencies.ETH));
 
     // Overflow is the balance of this project minus the amount that can still be distributed.
-    return _balanceOf < _ethDistributionRemaining ? 0 : _balanceOf - _ethDistributionRemaining;
+    return _balanceOf <= _ethDistributionRemaining ? 0 : _balanceOf - _ethDistributionRemaining;
```

Comment:
conceptually the same but we save ourselves from doing subtraction in the case balanceOf == _ethDistributionRemaining

Reply:
nice